code/__HELPERS/unsorted.dm

```diff
@@ -843,7 +843,7 @@ var/list/WALLITEMS_INVERSE = typecacheof(list(
 			if(id in hardcoded_gases || gas_concentration > 0.001) //ensures the four primary gases are always shown.
 				user << "<span class='notice'>[cached_gases[id][GAS_META][META_GAS_NAME]]: [round(gas_concentration*100, 0.01)] %</span>"
 
-		user << "<span class='notice'>Temperature: [round(air_contents.temperature-T0C)] &deg;C</span>"
+		user << "<span class='notice'>Temperature: [round(air_contents.temperature)] &deg;K</span>"
```

Comment:
On the contrary. This is the only device that still measures in celcius and its also the most convenient way to measure temperature, so why are we bombarding players with Kelvin just so we can throw celcius at them when they try to measure results. 

Its the year 2507, its time to have a uniform measure of temperature.